package.json

```diff
@@ -43,24 +43,23 @@
     "url": "git+https://github.com/welovecoding/website.git"
   },
   "lint-staged": {
-    "*.{json,js,jsx}": ["prettier --write", "git add"]
+    "*.{json,jsx}": ["prettier --write", "git add"]
   },
   "scripts": {
     "check": "nsp check --output summary",
     "coverage:backend":
       "rimraf coverage && npm run dist && cross-env JASMINE_CONFIG_PATH=src/test/support/jasmine.json istanbul cover --report html ./node_modules/jasmine/bin/jasmine.js",
-    "dist:frontend": "rimraf dist/frontend/js && webpack",
     "dist": "npm run init:db && npm run dist:frontend",
+    "dist:frontend": "rimraf dist/frontend/js && webpack",
     "init:db": "knex migrate:latest && knex seed:run",
-    "lint": "prettier --write '**/*.{json,js,jsx}'",
```

Comment:
you might want to keep that to be able to prettier everything right away.. (remove `json` of course if you don't want it now)